src/events/loginEvents.js

```diff
@@ -9,7 +9,8 @@ interview.AddEventsLogin = function () {
         if (usernameLogin === 'a' && passwordLogin === 'a') {
             sessionStorage.userLogged = 'Andreea';
             firstPage();
-        } else {
```

Comment:
I personally prefer the single line format for `} else {`, but if you like this better, then by all means, carry on! :wink:

Reply:
I prefer that too, but I used some "Fix all auto-fixable problems" who changed that and I didn't notice :P